learner_pathway_progress/management/tests/test_update_all_pathways_progress.py

```diff
@@ -54,15 +54,15 @@ def setUp(self):
 
     @patch("openedx.core.djangoapps.catalog.utils.check_catalog_integration_and_get_user")
     @patch("learner_pathway_progress.utilities.get_all_learner_pathways", )
-    def test_membership_and_progress_if_enrollment_exists(self, mocked_all_learner_pathways, mock_user):
+    def test_enterprise_membership_and_progress_if_enrollment_exists(self, mocked_all_learner_pathways, mock_user):
```

Comment:
can we somehow add the ability to pass enterprise uuids and test whether they are correctly placed in models, here in test?

Reply:
added